src/overlays/actors/ovl_En_Tg/z_en_tg.h

```diff
@@ -2,15 +2,35 @@
 #define Z_EN_TG_H
 
 #include "global.h"
+#include "objects/object_mu/object_mu.h"
 
 struct EnTg;
 
 typedef void (*EnTgActionFunc)(struct EnTg*, PlayState*);
 
+typedef struct EnTgHeartEffect {
+    /* 0x00 */ u8 isEnabled;
+    /* 0x01 */ u8 timer;
+    /* 0x02 */ char unk2[0x2];
+    /* 0x04 */ f32 scale;
+    /* 0x08 */ char unk8[0xC];
+    /* 0x14 */ Vec3f pos;
+    /* 0x20 */ Vec3f unusedZeroVec20;
```

Comment:
Others might disagree, but I think you can drop the `20` from this. The fact that it's at 0x20 in the struct is already evident by the comment next to it.